Model/.csv service/Csv Readers/CsvFlightReader.cs

```diff
@@ -0,0 +1,27 @@
+﻿using AirportTicketBooking.Model.Classes;
+using AirportTicketBooking.Model.csv_service.Class_map;
+using CsvHelper;
+using System.Globalization;
+
+namespace AirportTicketBooking.Model.csv_service.Csv_Readers;
+
+public class CsvFlightReader : ICsvReader<Flight>
+{
+    public List<Flight> Read()
+    {
+        using var streamReader = new StreamReader(Path.Combine(Environment.CurrentDirectory, $"flights.csv"));
```

Comment:
Can we move the files' names into a new file? so we can have a constant for each file name

Reply:
Like having a .csv file contains files names?